### ISSUE TITLE
Firefox logo

### ISSUE THREAD

**Issue by edent** [edent/SuperTinyIcons#204]
## The icon I want is:

* Name: Firefox
* URL: https://www.mozilla.org/en-GB/firefox/new/
* Link to icon: https://commons.wikimedia.org/wiki/File:Firefox_Logo,_2017.svg

Going to need @judeGibbons magic here! Even when I simplify the logo and remove the gradients, I can't get it even close.

I've already added Chrome, Opera, and Edge.

**Comment by judeGibbons** [comment]
I'll give it a go! Any gradients would have to be removed or massively simplified though.

**Comment by pluma** [comment]
Maybe try using the simplified version? https://design.firefox.com/product-identity/firefox-glyph/firefox-logo-glyph.svg

**Comment by edent** [comment]
Good shout! I've got it down to 1,02**7** bytes!

```
<svg xmlns="http://www.w3.org/2000/svg"
aria-label="Firefox" role="img"
viewBox="0 0 512 512"><rect
width="512" height="512"
rx="15%" fill="#fff"/><defs><linearGradient id="a" x1="90" x2="440" y1="418" y2="100" gradientUnits="userSpaceOnUse"><stop stop-color="#900"/><stop stop-color="#fc4" offset="1"/></linearGradient></defs><circle cx="256" cy="256" r="210" fill="#04e"/><path d="M489 236c1-22-15 23-9-18-7-46-37-82-65-117-9-13-14-28-17-44-6 9-3 38-16 19-19-17-44-34-47-61-32 38-52 95-27 141 20 46 77 61 96 108 9 21-3-4-10-11-9-11-20-21-33-29 29 61-12 142-80 151-36 7-84-9-99-40 36 16 69-6 101-17 31 4 1-34-17-30-23-3-45 18-67 3-16-8 18 3-11-9-5-3-30-25-17-43 10-12 14-6 26 3 2-4-10-14 2-4 8-12-3-22 4-27 16-12 50-14 51-41-11-15-39-3-52-18-23-15 15-40 20-51 24-9-28-2-12-6-19 5-33 14-45 21-25-9-60 1-69-24-5-12-2-37-14-15-6 12-13 17-18 41-1 39-38 65-41 106 3-8 28-51 7-3-5 20-8 41-8 62 0-1 12-58 6-3-3 96 90 185 165 206 53 15 115 17 164-12 20-18 32-37 60-46 28-15 40-49 40-76 24-32 29-85 29-116l2 2z" fill="url(#a)"/></svg>
```

**Comment by edent** [comment]
OK, 993, but the path isn't exactly perfect...

```
<svg xmlns="http://www.w3.org/2000/svg"
aria-label="Firefox" role="img"
viewBox="0 0 512 512"><rect
width="512" height="512"
rx="15%" fill="#fff"/><defs><linearGradient id="a" x1="90" x2="440" y1="418" y2="100" gradientUnits="userSpaceOnUse"><stop stop-color="#900"/><stop stop-color="#fc4" offset="1"/></linearGradient></defs><circle cx="256" cy="256" r="210" fill="#04e"/><path d="m480 218c-7-46-37-82-65-117-9-13-14-28-17-44-6 9-3 38-16 19-19-17-44-34-47-61-32 38-52 95-27 141 20 46 77 61 96 108 5 9-31-33-43-40 29 61-12 142-80 151-36 7-84-9-99-40 36 16 69-6 101-17 31 4 1-34-17-30-23-3-45 18-67 3 7 8-42-31-28-52 10-12 14-6 26 3 2-4-10-14 2-4 8-12-3-22 4-27 16-12 50-14 51-41-11-15-39-3-52-18-23-15 15-40 20-51 24-9-28-2-12-6-19 5-33 14-45 21-25-9-60 1-69-24-5-12-2-37-14-15-6 12-13 17-18 41-1 39-38 65-41 106 3-8 28-51 7-3-5 20-8 41-8 62 0-1 12-58 6-3-3 96 90 185 165 206 53 15 115 17 164-12 20-18 32-37 60-46 28-15 40-49 40-76 24-32 29-85 29-116-2-36-15 47-6-18z" fill="url(#a)"/></svg>
```

**Comment by judeGibbons** [comment]
I've got it to about 1160 with a more complex path (or 1130 with a not-so-good gradient) - I'm still working at it but I can do a version using the simplified single path if I don't manage to pare it down enough...

**Comment by judeGibbons** [comment]
Is it mandatory to include a gradient in a defs block? It works without...

**Comment by edent** [comment]
If it works, it works!

**Comment by edent** [comment]
Fixed in #205
